scripts/available_software/available_software.py

```diff
@@ -28,7 +28,7 @@
 EESSI_TOPDIR = "/cvmfs/software.eessi.io/versions/2023.06"
 
 # some CPU targets are excluded for now, because software layer is too incomplete currently
-EXCLUDE_CPU_TARGETS = ['aarch64/a64fx']
+EXCLUDE_CPU_TARGETS = ['aarch64/a64fx', 'aarch64/nvidia/grace']
```

Comment:
@trz42 Apparently this didn't work, see #416. We may need to do some work for the extra subdir, but for now we can maybe just exclude `aarch64/nvidia`?